src/gcovr/formats/gcov/parser/common.py

```diff
@@ -151,11 +155,28 @@ def check_hits(
 ) -> int:
     """
     Check if hits count is negative or suspicious, if the issue is ignored returns 0
-    >>> check_hits(1, "", {}, 10, {})
+    >>> check_hits(1, "code", {}, 10, {"location": ("file", 5)})
```

Comment:
The location isn't used.
```suggestion
    >>> check_hits(1, "code", {}, 10, {})
```

Reply:
Similar to [this comment](https://github.com/gcovr/gcovr/pull/1075#discussion_r2009019595).  
The key is parsed and required in `NegativeHits.raise_if_not_ignored` and `SuspiciousHits.raise_if_not_ignored`

Reply:
But here the function isn't called because we have a hit of `1` and therefore the data isn't used.

Reply:
I see. Let me remove these data